### PR TITLE
Replacement screen snippet tool

### PR DESCRIPTION
**Pull request by mattias-symphony**
https://perzoinc.atlassian.net/browse/SDA-1616

### CODE REVIEW

File: MakeAnnotations.h [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+
+static LRESULT CALLBACK makeAnnotationsWndProc( HWND hwnd, UINT message, WPARAM wparam, LPARAM lparam) {
+	struct MakeAnnotationsData* data = (struct MakeAnnotationsData*) GetWindowLongPtrA( hwnd, GWLP_USERDATA );
+	int const spaceForButtons = 50; // Reserve some pixels at the top for the buttons
+	switch( message ) {
+		case WM_COMMAND: {
+			// Handle button clicks
+			if( HIWORD( wparam ) == BN_CLICKED ) {
+				if( (HWND) lparam == data->doneButton ) {
+                    data->completed = TRUE;
```
Comment (johankwarnmarksymphony) [inline MakeAnnotations.h:91]: indent

**Review by johankwarnmarksymphony — APPROVED** [review]
indent

File: package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "screen-snippet",
+  "version": "0.0.0",
```
Comment (VishwasShashidhar) [inline package.json:3]: can you set the version to 1.0.0?

**Review by VishwasShashidhar — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (mattias-symphony) [reply package.json:3]: I'll do that once it is approved and we make a release

**Pull request merged**